retrieval/discovery/nerve.go

```diff
@@ -139,5 +140,9 @@ func parseNerveMember(data []byte, path string) (*model.LabelSet, error) {
 	labels[nerveEndpointLabelPrefix+"_port"] = model.LabelValue(fmt.Sprintf("%d", member.Port))
 	labels[nerveEndpointLabelPrefix+"_name"] = model.LabelValue(member.Name)
 
+	for k, v := range member.Labels {
+		labels[nerveEndpointLabelPrefix+"_labels_"+k] = model.LabelValue(v)
```

Comment:
Thanks for your comments. I don't see how we could extract tags from any format coming from ZK, wether it's ServerSet, nerve, go-nerve or anything.
Or maybe build a generic ZK service discovery, with regexes in the config file to extract the tags ?

And as a funny side note, what if I pushed the same feature into AirBnB's nerve in ruby ? that would make this PR acceptable :D